app/ui-angular/pom.xml

```
@@ -121,39 +124,47 @@
           </execution>
           <execution>
             <id>yarn ng lint</id>
+            <phase>compile</phase>
             <goals>
               <goal>yarn</goal>
             </goals>
             <configuration>
               <arguments>lint</arguments>
+              <yarnInheritsProxyConfigFromMaven>false</yarnInheritsProxyConfigFromMaven>
             </configuration>
           </execution>
           <execution>
             <id>yarn ng test</id>
+            <phase>compile</phase>
             <goals>
               <goal>yarn</goal>
             </goals>
             <configuration>
               <arguments>test:ci</arguments>
               <skip>${skipTests}</skip>
+              <yarnInheritsProxyConfigFromMaven>false</yarnInheritsProxyConfigFromMaven>
             </configuration>
           </execution>
           <execution>
             <id>yarn cleanup</id>
+            <phase>compile</phase>
```

I think this is used to remove any customized `config.json` files so they don't end up being packaged in the app, so doing this before probably makes more sense.

```suggestion
            <phase>generate-sources</phase>
```

---

Changed